pipenv/vendor/README.md

```diff
@@ -4,8 +4,20 @@ These packages are copied as-is from upstream to reduce Pipenv dependencies.
 They should always be kept synced with upstream. DO NOT MODIFY DIRECTLY! If
 you need to patch anything, move the package to `patched`.
 
-Known vendored versions:
+## Updatating Vendored Packages
 
-- python-dotenv: 0.8.2
+Requires:
+
+- [invoke](https://pypi.org/project/invoke/)
+
+- [pip](https://pypi.org/project/pip/)
+
+Modify the pip requirements file `vendor.txt`. Reference the [pip
+documenation](https://pip.pypa.io/en/stable/user_guide/#requirements-files).
+Execute the vendoring update script:
 
-When updating, update the corresponding LICENSE files as well.
+```$ invoke vendoring.update```
+
+## Known vendored versions:
+
+- python-dotenv: 0.8.2
```

Comment:
I'm not sure that there is a good argument for tracking package versions here, there is basically no chance anyone is going to keep that information up to date.